### PR TITLE
[bitnami/etcd] PodDisruptionBudget should be enabled by default

### LINKED ISSUE #9927: [bitnami/etcd] PodDisruptionBudget should be enabled by default
### Name and Version

bitnami/etcd

### What is the problem this feature will solve?

Without PDBs, every time a cluster is updated, since Kubernetes is not aware of how to handle the application without PDB, it may cause a majority failure. 

According to [the etcd docs on Majority Failure](https://etcd.io/docs/v3.5/op-guide/failures/#majority-failure):
> When the majority members of the cluster fail, the etcd cluster fails and cannot accept more writes.
>
>The etcd cluster can only recover from a majority failure once the majority of members become available. If a majority of members cannot come back online, then the operator must start disaster recovery to recover the cluster.

This means that manual intervention is needed.

### What is the feature you are proposing to solve the problem?

I see that this chart already has PDB configuration set, which is good. However I propose to have PDB be set to default true. Given that "we (chart developers)" are aware of the app we are deploying I don't think we ever want consumers of the chart to enable PDB. The reality is, those who haven't PDB enabled on the chart are most probably unaware how etcd handles a majority failure and if explained to them they will turn it on.

I am pretty new to helm charts so I am not sure if it is possible to automatically set PDB to the minimum of `(N-1)/2` of the replica count

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by cdemi**
**Description of the change**

Without PDBs, every time a cluster is updated, since Kubernetes is not aware of how to handle the application without PDB, it may cause a majority failure. 

According to [the etcd docs on Majority Failure](https://etcd.io/docs/v3.5/op-guide/failures/#majority-failure):
> When the majority members of the cluster fail, the etcd cluster fails and cannot accept more writes.
>
>The etcd cluster can only recover from a majority failure once the majority of members become available. If a majority of members cannot come back online, then the operator must start disaster recovery to recover the cluster.

This means that manual intervention is needed.

**Benefits**

Enabling this by default instructs the Kubernetes cluster and operators that at least 51% must remain available during voluntary disruptions, such as cluster upgrades.

**Possible drawbacks**
None as far as I can see

**Applicable issues**
  - fixes #9927

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by miguelaeh — COMMENTED** [review]
IMHO this change shouldn't be considered a patch, since upgrading a release will set the `pdb.create` to `true` in case you don't have it overridden in your `values.yaml` as well as the `pdb.minAvailable`.
Maybe a major change is too much, but even though it is not a feature addition, I think a minor bump would be better.
WDYT @carrodher ?

**Comment by cdemi** [comment]
I think it makes sense to upgrade the minor yes

**Comment by carrodher** [comment]
I think a minor is better yes, too much for a major

File: bitnami/etcd/Chart.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/bitnami-docker-etcd
   - https://coreos.com/etcd/
-version: 8.0.2
+version: 8.0.3
```
Comment (carrodher) [inline bitnami/etcd/Chart.yaml:27]:
```suggestion
version: 8.1.0
```

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Pull request merged**